doc/man3/SSL_CTX_set_client_hello_cb.pod

```diff
@@ -138,7 +138,7 @@ SSL_client_hello_get0_ciphers(), SSL_client_hello_get0_compression_methods(),
 SSL_client_hello_get0_ext(), and SSL_client_hello_get1_extensions_present()
 were added in OpenSSL 1.1.1.
 SSL_client_hello_get_extension_order()
-was added in OpenSSL 3.1.0.
+was added in OpenSSL 3.2.0.
```

Comment:
For consistency with everywhere else we really should make this just "3.2", not "3.2.0"

Reply:
Since this PR is only about search&replace, I would suggest treating that in a different PR afterwards.

Reply:
I agree with @mspncp, another PR